src/Pos/DB/DB.hs

```diff
@@ -104,7 +104,7 @@ initNodeDBs systemStart = do
 -- | Safely close all databases from 'NodeDBs'.
 closeNodeDBs :: MonadIO m => NodeDBs -> m ()
 closeNodeDBs NodeDBs {..} =
```

Comment:
Some functions in this module are specific to rocksdb and can be easily moved to `Pos.DB.Rocks` in `cardano-sl-db` I suppose.